src/source/Metrics/Metrics.c

```diff
@@ -0,0 +1,116 @@
+/**
+ * Kinesis WebRTC Metrics
+ */
+#define LOG_CLASS "Metrics"
+#include "../Include_i.h"
+
+STATUS getIceCandidatePairStats(PRtcPeerConnection pRtcPeerConnection, PRtcIceCandidatePairStats pRtcIceCandidatePairStats)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+    PKvsPeerConnection pKvsPeerConnection = (PKvsPeerConnection) pRtcPeerConnection;
+    CHK((pRtcPeerConnection != NULL || pRtcIceCandidatePairStats != NULL), STATUS_NULL_ARG);
+    DLOGW("getIceCandidatePairStats not supported currently");
```

Comment:
Also, add CHK(FALSE, STATUS_NOT_IMPLEMENTED);

Reply:
Same comments apply to the cases below

Reply:
Ah! Wasnt aware of this status code. Fixed everywhere